### ISSUE TITLE
Page does not start with a level 1 heading

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#649]
A level 1 heading `h1` tells the user what the page is about before they decide to navigate through the content. There still are pages on the website that do not start with `h1`. 

Siteimprove report: https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r61&ruleId=61&issueKind=1&exceptTags=1,2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

One of the causes for this issue is the [`h2` with an `id="midd-nav-label"` in office headers](https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=72112606362&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone#/sia-r61/failed/REc4OVVtZW8=;NjgxNGViNjc=).

**Comment by poojagunturu96** [comment]
Fix already in place in the frontend code for the issue in nav.twig, update needed in Drupal code.
